### PR TITLE
Shalamai.week1

### PR DESCRIPTION
**Pull request by shalamai**
This change is [Reviewable](https://reviewable.io/reviews/presly808/aco17/2)

### CODE REVIEW

**Comment by shalamai** [comment]
task completed, please cheack

File: VladShalamai/VladShalamai.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (presly808) [inline VladShalamai/VladShalamai.iml]: remove file from git directory
git rm --cached <file_name>

File: VladShalamai/src/Car.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * Created by ���� on 19.11.2016.
+ */
+public class Car {
+    private String model;
+    private int fuel;
+    private double price;
+    private boolean opened;
+    private int key = 123;
+
+    public void init(String model, int fuel, double price, boolean opened) {
```
Comment (presly808) [inline VladShalamai/src/Car.java]: replaced by constructor

File: VladShalamai/src/TestCargo.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * Created by ���� on 19.11.2016.
+ */
+public class TestCargo {
+    public static void main(String[] args) {
+        Car car = new Car();
+        car.init("bmw", 10, 5000, true);
+        int expFuel1 = 9;
+        int expFuel2 = 7;
+
+        car.go();
```
Comment (presly808) [inline VladShalamai/src/TestCargo.java]: put test logic to a particular method

File: VladShalamai/src/Car.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    public boolean getOpened() {
+        return opened;
+    }
+
+    public String showInfo() {
+        return String.format("model: %s, fuel: %d, price: %.2f, opened %b", model, fuel, price, opened);
+    }
+
+    public void go() {
+        fuel -= 1;
```
Comment (presly808) [inline VladShalamai/src/Car.java]: set varaibale (constant) for -1

File: VladShalamai/src/TestCarclose.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by ���� on 19.11.2016.
```
Comment (presly808) [inline VladShalamai/src/TestCarclose.java]: change windows encoding to utf8

**Comment by shalamai** [comment]
all done, please check)

**Comment by presly808** [comment]
good job
approved

**Pull request merged**